html-css/.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
How about we do just one more refactoring. Instead of copying the whole ```html-css``` folder to the root directory of our project, lets open the ```html-css``` folder and from there only copy the ```.github``` folder into the root directory of our project. Please let us do this, as that's the convention.

After this, let's ensure again. That all linter check are all passing 🟢.

Reply:
Added a new folder, checked for bugs, no bugs at least.